spec/helpers/react_helper_spec.rb

```diff
@@ -38,6 +39,54 @@
       end
     end
 
+    context 'when alert is present and has text and title' do
+      let(:alert) { instance_double(Alert, text: 'alert_title', title: 'alert_title') }
+
+      it 'includes alert and has text and title' do
+        helper.search_results_layout(search, {}, vertical, affiliate)
+        expect(helper).to have_received(:react_component).with(
+          'SearchResultsLayout',
+          hash_excluding(:alert)
```

Comment:
Did you mean to use `hash_including` instead? Because right now I don't think the test is testing what is saying it does.